torch/csrc/autograd/functions/basic_ops.cpp

```diff
@@ -20,7 +20,7 @@ auto DelayedError::apply(variable_list&& inputs) -> variable_list {
   outputs.reserve(inputs.size());
   for (auto& var : inputs) {
     // FIXME: share version counters
-    outputs.emplace_back(var.defined() ? var.data() : at::Tensor());
+    outputs.emplace_back(var.defined() ? var : at::Tensor());
```

Comment:
It's okay to remove `.data()` here, because the `wrap_outputs(...)` call below (implementation: https://github.com/pytorch/pytorch/blob/master/torch/csrc/autograd/functions/utils.cpp#L19 and https://github.com/pytorch/pytorch/blob/master/torch/csrc/autograd/functions/utils.cpp#L28) always calls `make_variable(...)` on each tensor in `outputs`, which ignores the tensor's original AutogradMeta and creates a new AutogradMeta.